okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -1201,7 +1201,7 @@ private static DeploymentDescriptor pickInstance(List<DeploymentDescriptor> inst
     Map<String, String> headers = sysReqHeaders(headersIn, tenantId, authToken, inst, modPerms);
     headers.put(XOkapiHeaders.URL_TO, inst.getUrl());
     logger.debug("syscall begin {} {}{}", inst.getMethod(), inst.getUrl(), inst.getPath());
-    OkapiClient cli = new OkapiClient(this.httpClient, inst.getUrl(), vertx, headers);
+    OkapiClient cli = new OkapiClient(httpClient.getHttpClient(), inst.getUrl(), vertx, headers);
```

Comment:
OkapiClient does not need this new `FuturisedHttpClient`?

Reply:
OkapiClient wraps HttpClient into WebClient, and WebClient wraps the call into a try-catch-block: https://github.com/vert-x3/vertx-web/blob/4.0.0/vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/HttpContext.java#L305